src/scripting/lua_kernel_base.cpp

```diff
@@ -471,22 +483,16 @@ lua_kernel_base::lua_kernel_base()
 	cmd_log_ << "Registering basic wesnoth API...\n";
 
 	static luaL_Reg const callbacks[] {
-		{ "compare_versions",         &intf_compare_versions         		},
 		{ "deprecated_message",       &intf_deprecated_message              },
-		{ "have_file",                &lua_fileops::intf_have_file          },
-		{ "read_file",                &lua_fileops::intf_read_file          },
-		{ "canonical_path",           &lua_fileops::intf_canonical_path     },
 		{ "textdomain",               &lua_common::intf_textdomain   		},
 		{ "dofile",                   &dispatch<&lua_kernel_base::intf_dofile>           },
 		{ "require",                  &dispatch<&lua_kernel_base::intf_require>          },
 		{ "kernel_type",              &dispatch<&lua_kernel_base::intf_kernel_type>          },
 		{ "compile_formula",          &lua_formula_bridge::intf_compile_formula},
 		{ "eval_formula",             &lua_formula_bridge::intf_eval_formula},
 		{ "name_generator",           &intf_name_generator           },
-		{ "random",                   &intf_random                   },
 		{ "log",                      &intf_log                      },
-		{ "get_image_size",           &intf_get_image_size           },
-		{ "get_time_stamp",           &intf_get_time_stamp           },
+		{ "ms_since_init",            &intf_get_time_stamp           },
```

Comment:
Why isn't the C++ function getting renamed?

Reply:
There's no particular reason, I just didn't bother. I can rename it if you prefer.

Reply:
Please keep names consistent across lua and C++ usage.

Reply:
Done